test/gmp/GeneralMessagePassing.js

```diff
@@ -106,9 +112,17 @@ describe('GeneralMessagePassing', () => {
         destinationChainGateway = await deployGateway();
         const constAddressDeployer = await deployContract(ownerWallet, ConstAddressDeployer);
 
-        sourceChainGasService = await deployUpgradable(constAddressDeployer.address, ownerWallet, GasService, GasServiceProxy, [
+        gasService = await deployUpgradable(constAddressDeployer.address, ownerWallet, GasService, GasServiceProxy, [ownerWallet.address]);
+
+        const expressProxyDeployer = await deployContract(ownerWallet, ExpressProxyDeployer, [destinationChainGateway.address]);
```

Comment:
We'll need to make some minor tweaks here to switch from waffle to hardhat, should be pretty easy though

Reply:
@deanamiel could you please try to convert this test file and remove all direct artifact imports?

Reply:
yup